pkg/kubelet/rkt/pod_info.go

```diff
@@ -49,9 +51,9 @@ type podInfo struct {
 	ip string
 	// The pid of the init process in the pod.
 	pid int
-	// A map from image hashes to exit codes.
-	// TODO(yifan): Should be appName to exit code in the future.
+	// A map of [app name]:[exit code].
 	exitCodes map[string]int
+	// TODO(yifan): Expose [app name]:[image id].
 }
 
 // parsePodInfo parses the result of 'rkt status' into podInfo.
```

Comment:
the output of rkt status keep changing, and I expects it is going to change in the future until we stabilize it. Without that output in the comment, it is very hard to review. For example the above question I raised, I guessed that is the case you just answered, but not sure about it. :-)